webapp/src/main.ts

```diff
@@ -1,8 +1,29 @@
 import Vue from 'vue';
 import App from './App.vue';
+// const path = require('path');
```

Comment:
Could this also be some forgotten variable?

Reply:
![Screenshot - 2020-11-09T183144 040](https://user-images.githubusercontent.com/2110676/98575728-d69f7280-22b9-11eb-80f0-d9354e4c0907.png)


@Marcelloloco use the suggestion button :point_up: 
```suggestion
```